.travis.yml

```diff
@@ -80,6 +80,22 @@ matrix:
         - go get github.com/bazelbuild/buildtools/buildifier
         - ./ci/travis/bazel-format.sh
 
+    - os: linux
+      env: SANITIZER=1 CC=clang PYTHON=3.5 PYTHONWARNINGS=ignore
+
+      install:
+        - eval `python $TRAVIS_BUILD_DIR/ci/travis/determine_tests_to_run.py`
```

Comment:
- Put quotes around `$TRAVIS_BUILD_DIR`.
- Use `$(...)` instead of backticks. (We should really migrate all of them at some point.)

Reply:
We should be consistent with the rest of the file, feel free to submit a PR to change it.